src/carousel/carousel.spec.ts

```diff
@@ -321,9 +322,53 @@ describe('ngb-carousel', () => {
        discardPeriodicTasks();
      }));
 
+  it('should listen to mouse events based on pauseOnHover attribute', fakeAsync(() => {
+
+       const html = `
+    <ngb-carousel [pauseOnHover]='pauseOnHover'>
+      <ng-template ngbSlide>foo</ng-template>
+      <ng-template ngbSlide>bar</ng-template>
+    </ngb-carousel>
+  `
+
+           const fixture = createTestComponent(html);
+
+       const carouselDebugEl = fixture.debugElement.query(By.directive(NgbCarousel));
+
+       expectActiveSlides(fixture.nativeElement, [true, false]);
+
+       carouselDebugEl.triggerEventHandler('mouseenter', {});
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [true, false]);
+
+       tick(6000);
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [true, false]);
+
+       carouselDebugEl.triggerEventHandler('mouseleave', {});
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [true, false]);
+
+       tick(6000);
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [false, true]);
+
+       fixture.componentInstance.pauseOnHover = false;
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [false, true]);
+
+       carouselDebugEl.triggerEventHandler('mouseenter', {});
+       fixture.detectChanges();
+
+       tick(6000);
+       fixture.detectChanges();
+       expectActiveSlides(fixture.nativeElement, [true, false]);
+       discardPeriodicTasks();
+     }));
+
   it('should pause / resume slide change with time passage on mouse enter / leave', fakeAsync(() => {
        const html = `
-      <ngb-carousel>
+      <ngb-carousel [pauseOnHover]='pauseOnHover'>
```

Comment:
I don't see the point about adding that here. It basically duplicate with your added test.
Both `'should listen to mouse events based on pauseOnHover attribute'` and `'should pause / resume slide change with time passage on mouse enter / leave'` are the same.